src/components/AddItem/AddItem.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Button, Modal, Form } from 'semantic-ui-react'
+
+const AddItem = () => {
+  const [open, setOpen] = React.useState(false)
+
+  return (
+    <Modal
+      onClose={() => setOpen(false)}
+      onOpen={() => setOpen(true)}
+      open={open}
+      trigger={<Button>Add Item</Button>}
+    >
+      <Modal.Header>Add Item</Modal.Header>
+      <Modal.Content>
+        <Form>
+          <Form.Group>
+            <Form.Input required label="Name" placeholder="Air Jordan 1 High Mocha" width={7} />
```

Comment:
Perhaps we can extract the width numbers to a constant where you can map each number to a semantic value such as "MEDIUM", "SMALL", "LARGE".

Reply:
This is called a "magic number".